scripts/weblate/weblate_locks_notify_community.py

```diff
@@ -17,18 +17,28 @@
 projects['playbooks']='playbooks/webapp'
 projects['glossary']='mattermost/glossary'
 
-w=wlc.Weblate(apikey.myAPIKey)
+mention={}
+mention['mobile']='cc: @guillermo.vaya,@zef.hemel'
+mention['server']='cc: @guillermo.vaya,@zef.hemel'
+mention['webapp']='cc: @guillermo.vaya,@zef.hemel'
+mention['focalboard']='cc: @jesus.espino,@scott.bishel'
+mention['playbooks']='cc: @jesse.hallam,@caleb.roseland'
+mention['glossary']=''
+
+w=wlc.Weblate(key=apikey.myAPIKey)
 current_locks={}
-previous_locks= json.load(open("locks.txt"))
+previous_locks= json.load(open("/home/tom/scripts/mattermost/locks.txt"))
```

Comment:
Will the `/home/tom/scripts/mattermost` path exist on the Weblate server? If not, this path may need to be modified.

Reply:
yeah, mixing two branches, one that is running at my local server and official one :facepalm: 
Also working on the train in a terminal with vi was not my best idea